conf/cassandra.yaml

```diff
@@ -1625,6 +1625,14 @@ drop_compact_storage_enabled: false
 # The two thresholds default to -1 to disable.
 # in_select_cartesian_product_warn_threshold: -1
 # in_select_cartesian_product_fail_threshold: -1
+# Guardrail to warn or fail when encountering larger size of collection data than threshold.
```

Comment:
I would explain here how the _fail_ only applies to fragments while it will only _log_ and _not fail_ at later stages.

Reply:
Makes sense, that behaviour deserves an explanation in config.